src/CalculatorUITests/StandardModeFunctionalTests.cs

```diff
@@ -0,0 +1,269 @@
+using CalculatorUITestFramework;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Remote;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+
+namespace CalculatorUITests
+{ 
+    [TestClass]
+    public class StandardModeFunctionalTests
+    {
+        private StandardOperatorsPanel standardOperators => StandardCalculatorPage.StandardOperators;
+
+        /// <summary>
+        /// Initializes the WinAppDriver web driver session.
+        /// </summary>
+        /// <param name="context"></param>
+        [ClassInitialize]
+        public static void ClassInitialize(TestContext context)
+        {
+            StandardCalculatorPage.StandardCalculatorSetup(context);
+        }
+
+        /// <summary>
+        /// Closes the app and WinAppDriver web driver session.
+        /// </summary>
+        [ClassCleanup]
+        public static void ClassCleanup()
+        {
+            StandardCalculatorPage.TearDown();
+        }
+
+        /// <summary>
+        /// Ensures the calculator is in a cleared state
+        /// </summary>
+        [TestInitialize]
+        public void TestInit()
+        {
+            StandardCalculatorPage.ClearAll(false);
```

Comment:
nit: explicit names for boolean parameters for readability: `.ClearAll(clearHistory: false)`